tests/testthat/test_indicator_native_range_year.R

```diff
@@ -0,0 +1,7 @@
+library(readr)
+data_input_checklist_indicators <- read_delim("https://raw.githubusercontent.com/trias-project/indicators/master/data/interim/data_input_checklist_indicators.tsv", 
+                                              "\t", escape_double = FALSE, trim_ws = TRUE)
+
+source("./R/indicator_native_range_year.r")
```

Comment:
You don't need and don't have to source within a package. Installing a package and loading it makes all functions available for tests as well.

Run `devtools::install()` and ´library(trias)` (the latter command needed the first time) after you changed something in the functions and you have all functions ready to be used.
So, in package development, `devtools::install()` is your best friend, `source` your enemy :smile:

Reply:
When I use devtools::install() with the script without plotly::layout() i get this error:
`Note: possible error in 'layout(xaxis = list(title = x_lab, ': unused arguments (xaxis = list(title = x_lab, tickangle = "auto"), yaxis = list(title = y_lab, tickformat = ",d"), margin = list(b = 80, t = 100), barmode = ifelse(nlevels(summaryData$first_observed) == 1, "group", "stack")) at indicator_native_range_year.R:104 `

I suspected this was due to R thinking layout from the package _graphics_ should be used, while the importFrom line clearly states  layout should come from the plotly package (`@importForm plotly ggplotly, layout`).
Adding `plotly::` before layout solves this problem.  Any idea why this is ?

Reply:
somehow rebuilding the NAMESPACE (found out some functions I imported were missing) does not include plotly to the list. Maybe this is the cause??